pkg/project/testoutput/openshift/ci-operator/build-image/Dockerfile

```diff
@@ -1,16 +1,17 @@
 # DO NOT EDIT! Generated Dockerfile.
 
+FROM registry.ci.openshift.org/ocp/4.17:cli-artifacts as tools
```

Comment:
For now this is OK since we support only amd64 but there should be `--platform=linux/amd64` because the registry.ci.openshift.org/ocp/4.17:cli-artifacts is not multiarch.

Reply:
This comment probably should have been on the template file.